website/docs/en/plugins/webpack/split-chunks-plugin.mdx

```diff
@@ -293,3 +293,24 @@ module.exports = {
   },
 };
 ```
+
+#### splitChunks.cacheGroups.\{cacheGroup\}.reuseExistingChunk
+
+- **Type:** `boolean`
+- **Default** `false`
+
+Whether to reuse existing chunks when possible. For example:
```

Comment:
```suggestion
Whether to reuse existing chunks when possible. If the current chunk contains modules already split out from the main bundle, it will be reused instead of a new one being generated. This can affect the resulting file name of the chunk. For example:
```